### PR TITLE
Enhance README

### PR DESCRIPTION
**Pull request by polina-popova**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+
+| Parameter    | Description |
+|------------|---------|
+| build target_directory | Setup all CusDeb services with all dependences to the 'target_directory'. It supposed to be empty.
+| start                  | Start all services.
+| stop                   | Stop all services.
+| remove                 | Delete all artefacts created by 'build' command.
+| create-superuser       | Create superuser to Django admin interface.
+| dbshell                | Connect to PostgreSQL via terminal.
+| makemigrations         | Generate migrations based on chandes in models.
```
Comment (eugulixes) [inline README.md]: There is a typo.
`changes`

File: README.md [hunk 2]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| migrate                | Implement all migrations.
```
Comment (eugulixes) [inline README.md]: Apply all migrations.

File: README.md [hunk 3]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| makemessages           | Create .po files from .html files and place them in dashboard/locale for translation.
```
Comment (eugulixes) [inline README.md]: Use ` to highlight the file extensions.

File: README.md [hunk 4]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build target_directory | Setup all CusDeb services with all dependences to the 'target_directory'. It supposed to be empty.
+| start                  | Start all services.
+| stop                   | Stop all services.
+| remove                 | Delete all artefacts created by 'build' command.
+| create-superuser       | Create superuser to Django admin interface.
```
Comment (eugulixes) [inline README.md]: Probably `for Django admin interface`?

File: README.md [hunk 5]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
 ```
-Project depends on some code which is not yet published in GitHub
+
+
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build target_directory | Setup all CusDeb services with all dependences to the 'target_directory'. It supposed to be empty.
```
Comment (denismosolov) [inline README.md]: i think that would be a little bit more clear if we replace `build target_directory` with `build <target_directory>`

File: README.md [hunk 6]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+
+When you start services you can specify the ports for them in variables DASHBOARD_PORT, BM_PORT and DOMINION_PORT. For example: 
+```
+sudo env DASHBOARD_PORT=3000 ./single-node.sh start
+``` 
+Default ports are:
+
+| Service    | Default |
+|------------|---------|
+| Dashboard  | 8000    |
```
Comment (denismosolov) [inline README.md]: Dashboard is 8001

File: README.md [hunk 7]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| BlackMagic | 8001    |
+| Dominion   | 8003    |
+    
+## Authors
+See [AUTHORS](https://github.com/tolstoyevsky/cusdeb-single-node/blob/master/AUTHORS.md).
```
Comment (denismosolov) [inline README.md:67]: I think we should add more people to AUTHORS.md

File: README.md [hunk 8]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+When you start services you can specify the ports for them in variables DASHBOARD_PORT, BM_PORT and DOMINION_PORT. For example: 
+```
+sudo env DASHBOARD_PORT=3000 ./single-node.sh start
+``` 
+Default ports are:
+
+| Service    | Default |
+|------------|---------|
+| Dashboard  | 8000    |
+| BlackMagic | 8001    |
```
Comment (denismosolov) [inline README.md]: bm is 8002

File: README.md [hunk 9]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build target_directory | Setup all CusDeb services with all dependences to the 'target_directory'. It supposed to be empty.
+| start                  | Start all services.
+| stop                   | Stop all services.
+| remove                 | Delete all artefacts created by 'build' command.
```
Comment (eugulixes) [inline README.md]: ```...by the `build` command.```

File: README.md [hunk 10]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| create-superuser       | Create superuser to Django admin interface.
+| dbshell                | Connect to PostgreSQL via terminal.
```
Comment (eugulixes) [inline README.md]: Connect to the project's database.

File: README.md (hunk 7 above)
Comment (polina-popova) [reply README.md:67]: We can do it but in the other PR.

File: README.md [hunk 11]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| dbshell                | Connect to the project's database.
+| makemigrations         | Generate migrations based on changes in models.
+| migrate                | Apply all migrations.
+| makemessages           | Create `.po` files from `.html` files and place them in dashboard/locale for translation.
+| compilemessages        | Comlipe `.po` files to `.mo` files
+| shell                  | Start Python interpreter with established enviroment.
+| restart                | Restart all services.
+| restart <service_name>   | Restart specified service. <service_name> can be 'blackmagic' or 'dominion'.
+
+When you start services you can specify the ports for them in variables DASHBOARD_PORT, BM_PORT and DOMINION_PORT. For example:
```
Comment (eugulixes) [inline README.md]:
```
When you start services you can specify the ports for them via the `DASHBOARD_PORT`, `BM_PORT` and `DOMINION_PORT` environment variables. For example
```

File: README.md [hunk 12]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+When you start services you can specify the ports for them in variables DASHBOARD_PORT, BM_PORT and DOMINION_PORT. For example: 
+```
+sudo env DASHBOARD_PORT=3000 ./single-node.sh start
+``` 
+Default ports are:
+
+| Service    | Default |
+|------------|---------|
+| Dashboard  | 8001    |
+| BlackMagic | 8002    |
```
Comment (eugulixes) [inline README.md]: Black Magic

File: README.md [hunk 13]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
 ```
-Project depends on some code which is not yet published in GitHub
+
+
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build <target_directory> | Setup all CusDeb services with all dependences to the <target_directory>. It supposed to be empty.
```
Comment (eugulixes) [inline README.md]: Use \` for `<target_directory>` in this and other cases.

File: README.md [hunk 14]
```diff
@@ -1,5 +1,9 @@
-# Installation
+# cusdeb-single-node 
+Script for making easier local deploying and debugging of CusDeb project.
```
Comment (eugulixes) [inline README.md]: The project is a set of scripts which is intended for simplifying deployment of CusDeb for developing purposes.

File: README.md [hunk 15]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+
+## Usage
+Script supports the following parameters:
+
+| Parameter    | Description |
+|------------|---------|
+| build <target_directory> | Setup all CusDeb services with all dependences to the <target_directory>. It supposed to be empty.
+| start                  | Start all services.
+| stop                   | Stop all services.
+| remove                 | Delete all artefacts created by `build` command.
```
Comment (eugulixes) [inline README.md]: Remove since the command is a stub right now.

File: README.md [hunk 16]
```diff
@@ -15,4 +19,41 @@ Build project in directory ../cusdeb.
...
+| create-superuser       | Create superuser for Django admin interface.
+| dbshell                | Connect to the project's database.
+| makemigrations         | Generate migrations based on changes in models.
+| migrate                | Apply all migrations.
+| makemessages           | Create `.po` files from `.html` files and place them in dashboard/locale for translation.
+| compilemessages        | Comlipe `.po` files to `.mo` files
+| shell                  | Start Python interpreter with established enviroment.
+| restart                | Restart all services.
+| restart <service_name>   | Restart specified service. <service_name> can be 'blackmagic' or 'dominion'.
```
Comment (eugulixes) [inline README.md]: Remove since the command is a stub right now.

**Review by eugulixes — APPROVED** [review]

**Pull request merged**